src/Commands/Site/Team/RoleCommand.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Pantheon\Terminus\Commands\Site\Team;
+
+use Pantheon\Terminus\Commands\TerminusCommand;
+use Pantheon\Terminus\Site\SiteAwareInterface;
+use Pantheon\Terminus\Site\SiteAwareTrait;
+use Terminus\Exceptions\TerminusException;
+
+class RoleCommand extends TerminusCommand implements SiteAwareInterface
+{
+    use SiteAwareTrait;
+
+    /**
+     * Change a team member's role.
+     *
+     * @command site:team:role
+     *
+     * @param string $site_id Site name to change roles on.
+     *
+     * @option string $member Email of the member to change.
+     * @option string $role Role to designate the member as.
+     *
+     * @usage terminus site:team:role my-site --member=admin@agency.com --role=admin
+     *   Change `admin@agency.com` to be role `admin` on the site `my-site`.
+     */
+    public function role($site_id, $options = ['member' => '', 'role' => ''])
```

Comment:
I think both member and role should be params here. `site:team:role <site> <member> <role>`